services/AuthService.tsでCannot find module エラー

起動時に下記エラーが発生しています
 ERROR  ERROR in services/AuthService.ts:2:26                                              00:05:43
TS2307: Cannot find module '../store/modules/auth.module' or its corresponding type declarations.
    1 | import axios from 'axios'
  > 2 | import { AuthUser } from '../store/modules/auth.module'
      |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

--- a/services/AuthService.ts
+++ b/services/AuthService.ts
@@ -1,5 +1,5 @@
 import axios from 'axios'
-import { AuthUser } from '../store/modules/auth.module'
+import { AuthUser } from '~/types/auth'
 
 const API_URL = 'https://monitoring.stopcovid19.jp/stg/api/nurse/'
 

